README.md

```diff
@@ -16,3 +16,34 @@ Command                                         | Description
 ```$ ./scripts/feeds/install device-manager```  | Install only the package device-manager
 
 
+### Building CreatorKit packages using pre-compiled OpenWrt SDK for Ci40 (Marduk):
+
+1. First download the SDK from https://downloads.creatordev.io/pistachio/marduk/OpenWrt-SDK-0.9.4-pistachio-marduk_gcc-5.2.0_musl-1.1.11.Linux-x86_64.tar.bz2
```

Comment:
Coming to think of it, the SDK version would just be for example, as it will keep changing.  How do we make this generic enough so that people can still get the latest SDK, and not the version specified here?